apps/condo/domains/notification/components/UserMessagesList/MessageCard/index.tsx

```diff
@@ -0,0 +1,71 @@
+import dayjs from 'dayjs'
+import React, { useCallback, useMemo } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+import { Card, Typography } from '@open-condo/ui'
+
+import { useTracking } from '@condo/domains/common/components/TrackingContext'
+import { MessageTypeAllowedToFilterType, UserMessageType } from '@condo/domains/notification/utils/client/constants'
+
+import './MessageCard.css'
+
+
+type MessageCardProps = {
+    message: UserMessageType
+    viewed?: boolean
+}
+
+const MESSAGE_ICON: Record<MessageTypeAllowedToFilterType, string> = {
+    PASS_TICKET_CREATED: '🔑',
+    TICKET_COMMENT_CREATED: '✏️',
+    TICKET_CREATED: '📬',
+}
+
+const DATE_FORMAT = 'DD.MM.YYYY, HH:mm'
+
+export const MessageCard: React.FC<MessageCardProps> = ({ message, viewed }) => {
+    const messageType = useMemo(() => message?.type, [message?.type])
+
+    const intl = useIntl()
+    const MessageTitle = intl.formatMessage({ id: `notification.UserMessagesList.message.${messageType}.label` })
+
+    const { logEvent } = useTracking()
+
+    const messageContent = useMemo(() => message?.defaultContent?.content, [message?.defaultContent?.content])
+    const titleLink = useMemo(() => message?.meta?.data?.url, [message?.meta])
```

Comment:
Can we guarantee that this link does not lead to a third-party site? Should we add some validation?

Reply:
Added `meta.data.url` check in `sendB2BAppPushMessage`